### ISSUE TITLE
Add keyboard navigation for color picker in table properties

### ISSUE THREAD

**Issue by mmotyczynska** [ckeditor/ckeditor5#12193]
## Provide a description of the task

Adding keyboard navigation to font color dropdown does not automatically cover color pickers in table properties - they have to be handled separately. This was checked during the PoC #11851.

*   It has to be possible to enter into color picker with the keyboard.
    *   So it should be possible to focus it with a tab when e.g. table cell properties dialog is open.
*   The colors need to act like a grid.
*   Focus cycler **within** color picker should act as expected.

## 📃 Other details

*   Browser: …
*   OS: …
*   CKEditor version: …
*   Installed CKEditor plugins: …
